homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -174,20 +162,11 @@ def __init__(
         vehicle: ConnectedDriveVehicle,
         description: BMWSensorEntityDescription,
         unit_system: UnitSystem,
-        service: str | None = None,
     ) -> None:
         """Initialize BMW vehicle sensor."""
         super().__init__(account, vehicle)
         self.entity_description = description
 
-        self._service = service
-        if service:
-            self._attr_name = f"{vehicle.name} {service.lower()}_{description.key}"
-            self._attr_unique_id = f"{vehicle.vin}-{service.lower()}-{description.key}"
-        else:
-            self._attr_name = f"{vehicle.name} {description.key}"
```

Comment:
We still want to set name and unique_id attributes.

Reply:
Oops. I have re-added them in https://github.com/home-assistant/core/pull/59881/commits/ca7676ed97dadba099487d3bdba9071bd58cdfa2. However it doesn't seem to be reflected in the PR?!?